src/plugins/orangehrmClaimPlugin/Dto/ClaimEventSearchFilterParams.php

```diff
@@ -23,7 +23,7 @@
 
 class ClaimEventSearchFilterParams extends FilterParams
 {
-    public const ALLOWED_SORT_FIELDS = ['claimEvent.name','claimEvent.status'];
+    public const ALLOWED_SORT_FIELDS = ['claimEvent.name','claimEvent.status','claimEvent.id'];
```

Comment:
```suggestion
    public const ALLOWED_SORT_FIELDS = ['claimEvent.name','claimEvent.status'];
```
remove `claimEvent.id`